pom.xml

```diff
@@ -41,10 +41,27 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
+                <!-- Holy grail of eclipse configuration:  https://stackoverflow.com/a/49599478/10707488   -->
                 <configuration>
+                    <compilerId>eclipse</compilerId>
                     <source>1.8</source>
                     <target>1.8</target>
+                    <compilerArguments>
+                        <properties>${basedir}/org.eclipse.jdt.core.prefs</properties>
+                    </compilerArguments>
                 </configuration>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.codehaus.plexus</groupId>
+                        <artifactId>plexus-compiler-eclipse</artifactId>
+                        <version>2.8.5</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.eclipse.jdt</groupId>
+                        <artifactId>ecj</artifactId>
+                        <version>3.20.0</version>
+                    </dependency>
+                </dependencies>
```

Comment:
Added eclipse compiler

Reply:
I cannot see any difference on the command line. Is the command still ' mvn clean compile assembly:single'?

Reply:
Yes the command is still the same. I believe the change doesn't reflect in the output messages. Could you change the warning flag to error on your local machine and try compiling? It should fail with errors triggered by eclipse compiler.

Reply:
@CanBerker changing the flag to error still does not output any errors. We can take a look at this in our next meeting.